packages/eslint-plugin/docs/rules/no-type-alias.md

```diff
@@ -118,6 +118,9 @@ type Foo = string | string[];
 
 type Foo = string & string[];
 
+// Template Literal Types
```

Comment:
Nit: we generally don't explain type names like this. The rest of the rule's docs don't.

```suggestion
```

Reply:
@JoshuaKGoldberg Sure. I've removed comments.